assets/javascripts/modules/analytics/setup.js

```diff
@@ -1,34 +1,37 @@
-/*global guardian:true */
 define([
     'utils/cookie',
     'modules/analytics/ga',
     'modules/analytics/omniture',
+    'modules/analytics/remarketing',
     'modules/analytics/krux',
     'modules/analytics/affectv'
 ], function (
     cookie,
     ga,
     omniture,
+    remarketing,
     krux,
     affectv
 ) {
     'use strict';
 
-    function init() {
-
-        var analyticsEnabled = (
-            guardian.analyticsEnabled &&
-            !navigator.doNotTrack &&
-            !cookie.getCookie('ANALYTICS_OFF_KEY')
-        );
+    var analyticsEnabled = (
+        window.guardian.analyticsEnabled &&
```

Comment:
> Are all these values set correctly when this module is loaded? i.e. was there definitely no reason for waiting until init() is called before checking them?

They get set correctly, but yeah this can just go inside init.